app/Models/Mship/Concerns/HasCTSAccount.php

```diff
@@ -26,8 +44,33 @@ public function syncToCTS()
 
             $ctsAccount = DB::table("{$ctsDatabase}.members")->where('cid', $this->id)->first();
 
+            // if no CTS account exists, lets sync to create a new one, and not update anything this time round.
             if (! $ctsAccount) {
-                // No user exists. Abort.
+                // for a division member, use the join timestamp of them joining, else use an empty timestamp.
+                // this is how CTS in its wisdom puts this column on visiting controllers.
+                $is_visitor = $this->primary_permanent_state->code != 'DIVISION';
+                $joined_div = ! $is_visitor
```

Comment:
Double check the live CTS - I feel like this should be null if the member is visiting (as they've never actually joined us). If it's not nullable, I would use the state start_at for the visiting row as the joined at.

Reply:
From what I could find, it is literally this empty type timestamp (gets written as something like 00-00-0000 00:00:00). It's a non-nullable column with no default.